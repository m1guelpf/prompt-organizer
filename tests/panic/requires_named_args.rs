use prompt_organizer::prompt;

prompt! {requires_named_args, "Hello, {}!"}

fn main() {}
