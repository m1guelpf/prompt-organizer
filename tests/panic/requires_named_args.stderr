error: proc macro panicked
 --> tests/panic/requires_named_args.rs:3:1
  |
3 | prompt! {requires_named_args, "Hello, {}!"}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Only named arguments are supported, e.g. {name}
